openshift_pipelines/op-release-notes.adoc

```diff
@@ -0,0 +1,48 @@
+//OpenShift Pipelines Release Notes
+include::modules/pipelines-document-attributes.adoc[]
+[id="op-release-notes"]
+= {pipelines-title} Release Notes
+:context: op-release-notes
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+{pipelines-title} is a cloud-native CI/CD experience based on the link:https://github.com/tektoncd[Tekton] project which provides:
+
+* Standard Kubernetes-native pipeline definitions (CRDs).
+* Serverless pipelines with no CI server management overhead.
+* Extensibility to build images using any Kubernetes tool, such as S2I, Buildah, JIB, and Kaniko.
+* Portability across any Kubernetes distribution.
+* Powerful CLI for interacting with pipelines.
+* Integrated user experience with the {product-title} Console developer perspective.
+
+////
+For an in-depth walk-through of {pipelines-title}, see xref:../openshift_pipelines/understanding-openshift-pipelines.adoc [Understanding OpenShift Pipelines].
+////
+[id="getting-support"]
+== Getting support
```

Comment:
Not critical, but I'd consider making the Getting support section it's own assembly/in the TOC. I know it's really small, but it doesn't really seem to be related to the release notes exactly.

Reply:
@bergerhoffer - I will take it as an enhancement for the next release. Hope that is ok with you. Thanks